src/node_buffer.cc

```diff
@@ -585,18 +585,19 @@ void Fill(const FunctionCallbackInfo<Value>& args) {
   }
 
   node::Utf8Value str(args.GetIsolate(), args[1]);
-  size_t str_length = str.length();
-  size_t in_there = str_length;
-  char* ptr = ts_obj_data + start + str_length;
+  const size_t str_length = str.length();
 
   if (str_length == 0)
     return;
 
+  char* ptr = ts_obj_data + start + str_length;
+
   memcpy(ts_obj_data + start, *str, MIN(str_length, length));
 
   if (str_length >= length)
     return;
 
+  size_t in_there = str_length;
```

Comment:
Other than making `str_length` the rest of these above just move variables around. Don't think that's necessary since the needed values can be easily calculated above.